test/role.test.js

```diff
@@ -47,9 +47,6 @@ describe('role model', function() {
     ACL.roleMappingModel = RoleMapping;
     ACL.userModel = User;
     ACL.applicationModel = Application;
-    Role.roleMappingModel = RoleMapping;
-    Role.userModel = User;
-    Role.applicationModel = Application;
```

Comment:
Nice!

I am wondering about the four lines above that set `ACL.*` models - are they indicating a similar problem in the `ACL` model as you are fixing in this patch? Is it something to eventually fix too?